parts/windowsparams.t

```diff
@@ -16,4 +16,11 @@
         "description": "Version of the Windows Server 2016 OS image to use for the agent virtual machines."
       },
       "type": "string"
-    }
\ No newline at end of file
+    },
+    "agentWindowsSourceUrl": {
```

Comment:
How is this different from WindowsImageSourceUrl?

Reply:
For now, it isn't except its in variables() rather than parameters().